node/src/components/consensus/highway_core/active_validator.rs

```diff
@@ -64,6 +69,10 @@ pub(crate) struct ActiveValidator<C: Context> {
     next_timer: Timestamp,
     /// Panorama and timestamp for a block we are about to propose when we get a consensus value.
     next_proposal: Option<(Timestamp, Panorama<C>)>,
+    /// The path to the file storing the hash of our latest known unit (if any).
+    unit_hash_file: Option<PathBuf>,
+    /// The hash of the last known unit created by us.
+    own_last_unit: Option<C::Hash>,
```

Comment:
So it's an `Option` type b/c of the tests? For production cases it's always expected to be `Some`?

Reply:
Also because when you start voting in an era, you don't have a last unit yet.